### PR TITLE
[office-js-preview] Updating Excel APIs

### PR DESCRIPTION
**Pull request by AlexJerabek**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://appsforoffice.microsoft.com/lib/beta/hosted/office.d.ts
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/office-js-preview/index.d.ts [hunk 1]
```diff
@@ -21878,7 +21933,28 @@ declare namespace Excel {
         set(properties: Interfaces.RangeUpdateData, options?: OfficeExtension.UpdateOptions): void;
         /** Sets multiple properties on the object at the same time, based on an existing loaded object. */
         set(properties: Excel.Range): void;
+        /**
+         *
+         * Fills range from the current range to the destination range.
+            The destination range must extend the source either horizontally or vertically. Discontiguous ranges are not supported.
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]: nit: this and later ones should include *

File: types/office-js-preview/index.d.ts [hunk 2]
```diff
@@ -21996,6 +22072,14 @@ declare namespace Excel {
          * @returns The Range which matched the search criteria.
          */
         findOrNullObject(text: string, criteria: Excel.SearchCriteria): Excel.Range;
+        /**
+         *
+         * Does FlashFill to current range.Flash Fill will automatically fills data when it senses a pattern, so the range must be single column range and have data around in order to find pattern.
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]:
```suggestion
         * Does FlashFill to current range. Flash Fill will automatically fills data when it senses a pattern, so the range must be single column range and have data around in order to find pattern.
```

**Comment by jessetrinity** [comment]
@ElizabethSamuel-MSFT are you still waiting for updates on this?

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Comment by ElizabethSamuel-MSFT** [comment]
@jessetrinity: Approved. Thanks.

**Pull request merged**